### PR TITLE
Add cassandra to table in readme

### PR DESCRIPTION
**Pull request by kragniz**
**What this PR does / why we need it**: add cassandra to table in readme

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,10 +29,11 @@ to others that are interested in our efforts & would like to see a new applicati
 Please search for or create an issue for the application in question you'd like to see a part of Navigator,
 and we can begin discussion on implementation & planning.
 
-| Name          | Version   | Status      | Notes                                                             |
-| ------------- | --------- | ----------- | ----------------------------------------------------------------- |
+| Name          | Version   | Status      | Notes                                                       |
+| ------------- | --------- | ----------- | ----------------------------------------------------------- |
 | Elasticsearch | 5.x       | Alpha       | [more info](docs/supported-types/elasticsearch-cluster.md)  |
-| Couchbase     |           | Coming soon |                                                                   |
+| Cassandra     | 12.x      | Alpha       | [more info](docs/supported-types/cassandra-cluster.md)      |
```
Comment (munnerz) [inline README.md]: Hmm.. not sure where cassandra 12.x has come from?

Comment (kragniz) [reply README.md]: I got it from [here](https://github.com/jetstack/navigator/blob/master/docs/quick-start/cassandra-cluster.yaml), but I now realise that's just the image version

**Pull request merged**